pages/staff/_slug.vue

```diff
@@ -35,8 +35,10 @@
             class="gothamist u-space--double--bottom mod-large"
             :show-gallery-icon="hasGallery(story.leadAsset)"
             :image="getArticleImage(story.leadAsset, story.ancestry[0].slug, story.listingImage)"
-            :image-height="150"
-            :image-width="150"
+            :image-height="560"
+            :image-width="413"
```

Comment:
Overall this looks good but the numbers for the layout sizes don't match the sizes i see used on the site:
Looking at the staff page I see
282x188px on desktop (not sure which breakpoint) 
650x433px on mobile (this shrinks down to fit the window width but that's the max width, mobile is bigger because we change to a vertical layout)

Reply:
let's hold on this, I was looking at the image cropping ticket and maybe the dimensions we've been using are wrong